src/include/cephfs/types.h

```diff
@@ -375,6 +375,9 @@ struct inode_t {
    */
   using client_range_map = std::map<client_t,client_writeable_range_t,std::less<client_t>,Allocator<std::pair<const client_t,client_writeable_range_t>>>;
 
+  static const uint8_t F_EPHEMERAL_DISTRIBUTED_PIN = (1<<0);
+  static const uint8_t F_QUIESCE_BLOCK             = (1<<1);
```

Comment:
I was thinking something like "barrier" but settled on "block". I don't think it's that confusing :man_shrugging:.

Reply:
if I want to exclude my directory from quiesce, I'd be contemplating whether to set a "quiesce.block" attribute on it, thinking it might prevent the quiesce rather than just ignore it

Reply:
> if I want to exclude my directory from quiesce, I'd be contemplating whether to set a "quiesce.block" attribute on it, thinking it might prevent the quiesce rather than just ignore it

But it does prevent quiesce for the entire tree (whereas before it did not when we traversed all inodes in the `SnapRealm`)? So I think in that way `block` does make sense.

Reply:
In the code we refer to it as "skip" and that would apply to the whole subtree if that attribute was set on a dir. I thought that the user would prefer to think about it the same way, as in "i don't want the quiesce to affect this file/dir".